__tests__/humanize.spec.js

```diff
@@ -45,10 +45,18 @@ describe('compactInteger tests', () => {
 
 describe('Ordinal value of numbers Test Suite', () => {
   describe('Ordinal value for numbers ending in zero', () => {
-    it('should return 0 if the number is 0 (cos 0th doesnt read very well)', () => {
+    it('should return 0 if the number is 0', () => {
       expect(Humanize.ordinal(0)).toEqual(0);
     });
 
+    it('should return the input if the number is 0', () => {
+      expect(Humanize.ordinal('0.')).toEqual('0.');
```

Comment:
It would make sense to me, since `ordinal(1.foo)` returns `1st`, not `1.foost`. It would however be somewhat of an backwards incompatible change, so I'll shoot in a separate PR for that